mteb/tasks/Image/ImageTextPairClassification/AROVisualAttribution.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from mteb.abstasks.Image.AbsTaskImageTextPairClassification import (
+    AbsTaskImageTextPairClassification,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class AROVisualAttribution(AbsTaskImageTextPairClassification):
+    images_column_names = ["image"]
+    texts_column_names = ["true_caption", "false_caption"]
+
+    metadata = TaskMetadata(
+        name="AROVisualAttribution",
+        description="Compositionality Evaluation of images to their captions.",
+        reference="https://openreview.net/forum?id=KRLUvxh8uaX",
+        dataset={
+            "path": "gowitheflow/ARO-Visual-Attribution",
+            "revision": "18f7e01358d91df599d723f00e16a18640e19398",
+        },
+        type="ImageTextPairClassification",
+        category="i2t",
+        eval_splits=["test"],
+        eval_langs=["eng-Latn"],
+        main_score="text_acc",
+        date=(
+            "2022-01-01",
+            "2022-12-31",
+        ),  # Estimated range for the collection of data
+        form=["written"],
+        domains=["Encyclopaedic"],
+        task_subtypes=["Caption Pairing"],
+        license="Not specified",
+        socioeconomic_status="mixed",
```

Comment:
We should have removed this one on main (as well as e.g. form). We should probably do an update soon

Reply:
> We should have removed this one on main (as well as e.g. form). We should probably do an update soon

I have removed form and socioeconomic status and will reflect in the next commit. Turns out some image datasets had them and we didn't remove them. For these 3, I copied the template from a task under the same task and forgot to remove!